website/addons/osfstorage/routes.py

```diff
@@ -112,6 +112,61 @@
             views.osfstorage_copy_hook,
             json_renderer,
         ),
+
+        Rule(
```

Comment:
You might consider doing a GET  `/project/<pid>/osfstorage/checkouts/` if there's a use to listing all of the files that are checked out. There would almost certainly be such a thing in API V2.